k8sdeploy.sh

```diff
@@ -0,0 +1,34 @@
+#!/bin/sh
+function _k8sdeploy() {
+  usage="k8sdeploy -- build, tag and push image to ecr
```

Comment:
`build, tag and deploy to k8s environment`?

Reply:
copy paste 💥 - cheers